.github/workflows/semgrep.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/checkout@v3
 
       - id: semgrep
-        run: semgrep ci --config=p/owasp-top-ten --config=p/cwe-top-25 --config=p/gitleaks -q --exclude="tests" --exclude="*/tests" --skip-unknown-extensions --suppress-errors
+        run: semgrep ci --config=p/owasp-top-ten --config=p/cwe-top-25 --config=p/gitleaks --config .semgrep/rules/detected-aws-account-id-in-arn.yaml --config r/generic.secrets.security.detected-aws-account-id.detected-aws-account-id --config r/generic.secrets.security.detected-aws-secret-access-key.detected-aws-secret-access-key -q --skip-unknown-extensions --suppress-errors
```

Comment:
We are disabling sending metrics to semgrep in the pre-commit and bitbucket pipelines, maybe we should do it here as well (`--metrics=off`).
Also, in the otm-aws-generator, we are excluding the generic.secrets.gitleaks.generic-api-key.generic-api-key rule, whereas we don't here, is that because of some reason?

Reply:
About the disabled metrics, the reason to have disabled in the pre-commit and bitbucket was for performance reasons, whereas in the github action we could assume that. However I disabled it.

About the generic-api-key rule. There is no reason to exclude it here.
IMO our goal is not to have exactly equal all the semgrep scans. This is run in github action and is public, the another is private an run in bitbucket pipelines. Each one has its own features, file contents and needs.
Let's discuss in the private repo if this rule fits our needs on it.